include/pagmo/problems/hock_schittkowski_71.hpp

```diff
@@ -59,7 +59,7 @@ namespace pagmo
  * Lecture Notes in Economics and Mathematical Systems, 187, 1981. doi: 10.1007/978-3-642-48320-2.
  *
  */
-struct PAGMO_DLL_PUBLIC hock_schittkowsky_71 {
+struct PAGMO_DLL_PUBLIC hock_schittkowski_71 {
```

Comment:
The reference on line 58 is to:
> * See: W. Hock and K. Schittkowski. Test examples for nonlinear programming codes.

https://link.springer.com/book/10.1007/978-3-642-48320-2

I couldn't see any explanation of why to reference w/ a `-y`.

Google has this to say when I try:
> Showing results for [Hock Schittkowski](https://www.google.com/search?sxsrf=ALiCzsYPY852I44O_A7g_uIwqjjcpM_Drg:1656305218904&q=Hock+Schittkowski&spell=1&sa=X&ved=2ahUKEwiMkNnh6cz4AhURb80KHfCJDB8QkeECKAB6BAgBEDM)
> Search instead for [Hock Schittkowsky](https://www.google.com/search?sxsrf=ALiCzsYPY852I44O_A7g_uIwqjjcpM_Drg:1656305218904&q=Hock+Schittkowsky&nfpr=1&sa=X&ved=2ahUKEwiMkNnh6cz4AhURb80KHfCJDB8QvgUoAXoECAEQNA)